lib/view_model/migrator.rb

```diff
@@ -5,16 +5,16 @@ class Migrator
     EXCLUDE_FROM_MIGRATION = '_exclude_from_migration'
 
     class << self
-      def migrated_deep_schema_version(viewmodel_class, required_versions, include_referenced: true)
+      def migrated_deep_schema_version(viewmodel_class, client_versions, include_referenced: true)
         deep_schema_version = viewmodel_class.deep_schema_version(include_referenced: include_referenced)
 
-        if required_versions.present?
+        if client_versions.present?
           deep_schema_version = deep_schema_version.dup
 
-          required_versions.each do |required_vm_class, required_version|
+          client_versions.each do |required_vm_class, client_version|
```

Comment:
I buy that argument. It's not quite as bad as suggested, since the old and new versions were both used in the cache keys so it already was being invalidated no matter what, but I think it just is less surprising to have the name-at-the-time in the structure produced here.